tmva/tmva/src/LossFunction.cxx

```diff
@@ -41,7 +41,8 @@ Huber Loss Function.
 
 #include "Rtypes.h"
 #include "TMath.h"
-
+#include "TStopwatch.h"
```

Comment:
I accidentally left these includes in. It's a relic from bench marking the timing vs nthreads.